mosaic-site/README.md

```diff
@@ -0,0 +1,29 @@
+# Local development
+
+Please follow these steps in order to setup your local environment.
+
+## Create a Github token
+
+On your Github profile, select `Settings` &#8594; `Developer settings` &#8594; `Personal access tokens` &#8594; `Tokens (classic)`.
```

Comment:
I believe it is (James and I had to do it) but I can double check with the Mosaic team and delete this later if it's not required

Reply:
Yeah. I wasn't able to run `yarn serve` successfully until I did that.

(Though, it would be good if Mosaic could be configured to avoid the need for this. Not sure why it needs to talk to Gtihubfor local dev work at all, TBH)